templates/default/components/MDX.js

```diff
@@ -2,9 +2,18 @@ import Head from 'next/head'
 import DemoComponent from './Demo'
 import { Pre } from './Pre'
 
+const Paragraph = (props) => {
```

Comment:
Why is this necessary? Looking at https://github.com/remarkjs/remark-math it seemed we could just use the remark plugins.

I'm guessing this is styling related. If so some comment here would have been recommended to explain this briefly.

Also if we used katex would this still be necessary?

And could we use traditional "css" for this? 

I ask as this seems a bit hacky and impinges on core code.

Reply:
Yes, this was style related. Moved to css and removed the component.